os-config.yml

```diff
@@ -37,7 +37,7 @@ rancher:
       - /lib/modules:/lib/modules
       - /lib/firmware:/lib/firmware
   bootstrap_docker:
-    args: [-d, -s, overlay, -b, none, --restart=false, -g, /var/lib/system-docker,
```

Comment:
We need --restart=false.  I know it's deprecated but we need this.  This is so that autorestart containers do start on system start.